packages/kbn-search-response-warnings/kibana.jsonc

```diff
@@ -1,5 +1,10 @@
 {
   "type": "shared-browser",
   "id": "@kbn/search-response-warnings",
-  "owner": "@elastic/kibana-data-discovery"
+  "owner": [
+    "@elastic/kibana-data-discovery"
+  ],
+  // FIXME? many platform plugins depend on it (for WarningHandlerCallback, handleWarnings, ...)
```

Comment:
Also curious about the FIXME here.

Reply:
Same as https://github.com/elastic/kibana/pull/199630#discussion_r1847872062